src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,31 @@
+package duke;
+
+import duke.storage.Storage;
+
+/**
+ * Duke is entry point of our program
+ */
+public class Duke {
+    public static final String LISTPATH = "./data/list.txt";
```

Comment:
I like how you extracted this out as a final variable instead of hardcoding each line with the file path